### DISCUSSION TITLE
Table onUpdate:pagination problem

### DISCUSSION THREAD

**Discussion by sasgosh** [quasarframework/quasar#10318] [General - Components / Directives / etc]
Hello,

I want to use **onUpdate:pagination** event to get page change ant that's working. But, every time when change page I get console message:

`[Vue warn]: Invalid prop: type check failed for prop "onUpdate:pagination". Expected Function, got Array`

So it works but with warn. Does anyone know why this occurs?

I made a simplified example at https://codepen.io/SaleCar/pen/dyWQZvj

**Comment by ontwikkelfabriek** [comment]
@SaleCar 

I took a look at it. 

I would change the pagination event handler function to this:
```javascript
pageChanged: (newPagination) => {
         console.log(newPagination)
}, 
```

About the warning, that looks very strange/wrong to me. I would create a github issue for this.

**Reply by sasgosh** [reply]
Yes, I have tried with newPagination, but it's the same [Vue warn] every time. It works, but it’s annoying. I'll wait a little to see if anyone has an idea. Thanks.

**Reply by metalsadman** [reply]
it's just a warning tho, and will not show in production.
